### ISSUE TITLE
deduplicate: each real life customer I want only have one record in the system. 

### ISSUE THREAD

**Issue by evaklopp** [buzzn/scrum#134]
**User story**
As PhO
I want for each real life customer I want only have one record in the system. 

---

**Definition of done**

- [ ] all duplication cases based on email and phonenumber have been reviewed 
- [ ] a real life customer exists only once in our system
- [ ] define the rules. within role like PT, PG, organisation the person/organisation exists only once

---
**XD**

What happens, if a person has more than one role? All combination of roles are possible. 

**Technical details**

_Frontend_

- [ ] ...

_Backend_

- [ ] ...

**Comment by evaklopp** [comment]
duplicate of #132 , closing
